lab_2_tokenize_by_bpe/main.py

```diff
@@ -14,10 +14,32 @@ def prepare_word(
     :param end_of_word: a token that signifies the end of word
     :return: preprocessed word
     """
+    if not isinstance(raw_word, str):
+        return None
+    if not isinstance(start_of_word, str) and start_of_word is not None:
+        return None
+    if not isinstance(end_of_word, str) and end_of_word is not None:
+        return None
+
+    symbol_list = []
+    for symbol in raw_word:
+        symbol_list.append(symbol)
+
+    if start_of_word is None and end_of_word is None:
+        return tuple(raw_word)
+    elif start_of_word is None and end_of_word is not None:
+        symbol_list.append(end_of_word)
+        return tuple(symbol_list)
+    elif start_of_word is not None and end_of_word is None:
+        symbol_list.insert(0, start_of_word)
+        return tuple(symbol_list)
+    symbol_list.insert(0, start_of_word)
+    symbol_list.append(end_of_word)
+    return tuple(symbol_list)
```

Comment:
I believe 4 return statements here could be conbined into one

Reply:
fixed